Source/CesiumRuntime/Public/CesiumTileExcluder.h

```diff
@@ -0,0 +1,75 @@
+#pragma once
+#include "CesiumTile.h"
+#include "CoreMinimal.h"
+#include "CesiumTileExcluder.generated.h"
+
+class CesiumTileExcluderAdapter;
+
+/**
+ * An actor component for excluding Cesium Tiles.
+ * This class provides an interface for excluding Cesium Tiles from a tileset.
+ * You can create a blueprint that derives from this class and override the
+ * `ShouldExclude` function to implement custom logic for determining whether a
+ * tile should be excluded. This function can be implemented in either C++ or
+ * Blueprints.
+ */
+UCLASS(
+    ClassGroup = (Cesium),
+    meta = (BlueprintSpawnableComponent),
+    Blueprintable,
+    Abstract)
+class CESIUMRUNTIME_API UCesiumTileExcluder : public UActorComponent {
+  GENERATED_BODY()
+private:
+  CesiumTileExcluderAdapter* pExcluderAdapter;
+
+protected:
+  UPROPERTY()
+  UCesiumTile* CesiumTile;
```

Comment:
Seems like this could be private, right? I think a Blueprint client can't use it anyway, because it's not marked BlueprintReadable or BlueprintReadWrite. And a C++ derived class probably has no reason to use it.